templates/helm/templates/leader-election-role-binding.yaml.tpl

```
@@ -14,6 +14,6 @@ roleRef:
   name: {{.ServicePackageName}}-leader-election-role
 subjects:
 - kind: ServiceAccount
-  name: {{.ServiceAccountName}}
+  name: {{ "{{ include "service-account.name" . }}" }}
```

Looks like we need to prepend an `\` for the helm brackets. I still get confused with this templinception..
```suggestion
  name: {{ "{{ include \"service-account.name\" . }}" }}
```

---

Hahaha, what a coincidence, I just pushed the same patch!